app/models/pydantic/query.py

```diff
@@ -11,9 +12,15 @@ class QueryRequestIn(StrictBaseModel):
 
 
 class QueryBatchRequestIn(StrictBaseModel):
-    feature_collection: Optional[FeatureCollection]
-    uri: Optional[str]
-    id_field: str
+    feature_collection: Optional[FeatureCollection] = Field(
+        None, description="An inline collection of GeoJson features on which to do the same query"
+    )
+    uri: Optional[str] = Field(
+        None, description="URI to a CSV or GeoJson file giving a list of features on which to do the same query.  For a CSV, the column with the geometry in WKB format should be named 'WKT'"
+    )
+    id_field: str = Field(
+        "fid", description="Name of field with the feature id, for use in labelling the results for each feature"
+    )
```

Comment:
Maybe underscore the fact that the field should contain a unique value?

Reply:
Good suggestion, changed the text to make that clearer.  Thanks!